Sell/Tests/SellImplementingTests.swift

```diff
@@ -14,13 +14,13 @@ final class SellImplementingTests: XCTestCase {
 
     func test_createOrderingJourneyViewController() {
         let graphQLManager = MockGraphQLManager<ApolloType.GetProductsQuery.Data>()
-        let sut = SellFactory.makeSellModule(graphQLManager: graphQLManager)
+        let sut = SellFactory.makeSellModule(graphQLManager: graphQLManager, colorStore: EmptyColorStore())
         XCTAssertTrue(sut.createOrderingJourneyViewController() is OrderingJourneyViewController)
     }
 
     func test_createOrderingJourneyView() {
         let graphQLManager = MockGraphQLManager<ApolloType.GetProductsQuery.Data>()
-        let sut = SellFactory.makeSellModule(graphQLManager: graphQLManager)
+        let sut = SellFactory.makeSellModule(graphQLManager: graphQLManager, colorStore: EmptyColorStore())
```

Comment:
Could you reuse `graphQLManager` and `sut` for these two test cases?

Reply:
Resolved in "improve coverage PR"